.env-example

```diff
@@ -12,7 +12,10 @@ OPENAI_API_KEY=sk-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 MAX_MODEL_TOKENS=4096
 
 # Whether or not to use prefixes !gpt and !dalle
-PREFIX_ENABLED=true
+PREFIX_ENABLED=false
```

Comment:
Enable PREFIX by default. Thats still the way to go since users will not understand that this bot decides automatically which model to use.

AUTO_MODEL is also 2x the cost